### PR TITLE
Add support for etcd ca rotation

### PR DESCRIPTION
**Pull request by asalkeld**
Jira AZURE-198

```release-note
Microsoft: Add geneva actions for rotating certificates and certificatesAndSecrets.
```
The current implementation just rotates etcd certificates and support for the other certs will be added soon.

### CODE REVIEW

File: pkg/config/v6/generate.go [hunk 1]
```diff
@@ -440,6 +440,12 @@ func (g *simpleGenerator) InvalidateSecrets() (err error) {
 	g.cs.Config.Certificates.GenevaLogging = api.CertKeyPair{}
 	g.cs.Config.Certificates.GenevaMetrics = api.CertKeyPair{}
 
+	// Angus TODO - check
```
Comment (asalkeld) [inline pkg/config/v6/generate.go]: this might need to go to a new function..

**Comment by asalkeld** [comment]
so the unit tests fail, but i don't really want to fix until I know where this fits (API).

**Comment by charlesakalugwu** [comment]
upgrade-v5.1 flake #1628 
/retest

**Comment by asalkeld** [comment]
Thanks @charlesakalugwu for the review, i'll get some fixes posted soon

**Comment by asalkeld** [comment]
/retest
flake https://github.com/openshift/openshift-azure/issues/1297

**Comment by asalkeld** [comment]
flake https://github.com/openshift/openshift-azure/issues/1438
/retest

**Pull request merged**